.github/scripts/trymerge.py

```diff
@@ -2043,7 +2039,7 @@ def handle_exception(e: Exception, title: str = "Merge failed") -> None:
         else:
             print("Missing comment ID or PR number, couldn't upload to Rockset")
     finally:
-        pr.remove_label(MERGE_IN_PROGRESS_LABEL)
+        gh_remove_label(org, project, args.pr_num, MERGE_IN_PROGRESS_LABEL)
```

Comment:
Maybe wrap `gh_remove_label` in a separate try for the case it doesn't exist?

Reply:
See description, this is done in `finally` block, so it should always exist at this point. And if it does not, then I would like to see the error (which would not be reported back to the user anyway)

Reply:
(Though, looks like I have to add it in order to pass testing, which reasonably does not have a token)